src/editors/sharedComponents/ErrorBoundary/ErrorPage.jsx

```diff
@@ -1,48 +1,88 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import {
   Button, Container, Row, Col,
 } from '@edx/paragon';
 
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import messages from './messages';
+import { navigateTo } from '../../hooks';
+import { selectors } from '../../data/redux';
 
 /**
  * An error page that displays a generic message for unexpected errors.  Also contains a "Try
  * Again" button to refresh the page.
  */
 export const ErrorPage = ({
   message,
+  studioEndpointUrl,
+  courseId,
+  // redux
+  unitData,
   // injected
   intl,
-}) => (
-  <Container fluid className="py-5 justify-content-center align-items-start text-center">
-    <Row>
-      <Col>
-        <p className="text-muted">
-          {intl.formatMessage(messages.unexpectedError)}
-        </p>
-        {message && (
-          <div role="alert" className="my-4">
-            <p>{message}</p>
-          </div>
-        )}
-        <Button onClick={global.location.reload()}>
-          {intl.formatMessage(messages.unexpectedErrorButtonLabel)}
-        </Button>
-      </Col>
-    </Row>
-  </Container>
-);
+}) => {
+  const courseOutlineUrl = `${studioEndpointUrl}/course/${courseId}`;
```

Comment:
what happens when we are editing a library? probably better to use terms like `learningContextId` and things like that.

Reply:
The problems here are twofold:
1. we should rename so it is learning context agnostic.
2. We should make sure returning to the "course" page works for a library